packages/wonder-blocks-core/util/types.ts

```diff
@@ -1,14 +1,16 @@
 // @flow
-import * as React from "react";
+import React, {MouseEvent, KeyboardEvent, TouchEvent, FormEvent, FocusEvent, ReactNode} from "react";
 
-import type {CSSProperties} from "aphrodite";
+import {OpenCSSProperties} from "aphrodite";
 
-type NestedArray<T> = $ReadOnlyArray<T | NestedArray<T>>;
 type Falsy = false | 0 | null | void;
+
+interface NestedArray<T> extends ReadonlyArray<T | NestedArray<T>> { };
```

Comment:
TypeScript requires the use of interfaces when writing recursive types, see https://github.com/Microsoft/TypeScript/issues/3496#issuecomment-128553540.